adhocracy4/comments/services.py

```diff
@@ -0,0 +1,8 @@
+from .models import Comment
+
+
+def delete_comments(contenttype, pk):
+    comments = Comment.objects.all().filter(
+        content_type=contenttype, object_pk=pk)
+    for comment in comments:
+        comment.delete()
```

Comment:
signaly.py registers a delete handler for the post_delete signal.

However, @slomo has created a generic function for this:
`generics.setup_delete_signals(settings.A4_COMMENTABLES, Comment)`

I'll create a pull request.